solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClient.java

```diff
@@ -330,6 +333,12 @@ protected ModifiableSolrParams calculateQueryParams(Set<String> queryParamNames,
     }
     return queryModParams;
   }
+  
+  private String changeV2RequestEndpoint(String basePath) throws MalformedURLException {
```

Comment:
this could be made a `public static` method

Reply:
This is a widely used client class, so we should probably be slightly more wary of adding public methods.  Perhaps package protected static... until we need it to be public for some reason.  I'll make that change when I commit.